classes/navigationMenu/NavigationMenuItemAssignmentDAO.inc.php

```diff
@@ -110,13 +110,35 @@ public function getByMenuIdAndParentId($navigationMenuItemId, $menuId, $parentId
 
 		$returner = null;
 		if ($result->RecordCount() != 0) {
-				$returner = $this->_fromRow($result->GetRowAssoc(false));
+			$returner = $this->_fromRow($result->GetRowAssoc(false));
 		}
 		$result->Close();
 		return $returner;
 	}
 
 	/**
+	 * Retrieve items by navigationMenuItemId menu item id and ParentId
```

Comment:
Update the description of this function.

Reply:
Done